### DISCUSSION TITLE
Whisper bug

### DISCUSSION THREAD

**Discussion by MX-05** [openai/whisper#966] [Q&A]
Hi, i'm using whisper to make subtitles for my family's videos. But sometimes whisper write n times the same phrase. to rewrite this phrase whisper lost some audio's words.

To make it faster i'm using google colab with this code blocks

```
!pip install torch # to check if cuda gpu is enabled

# install whisper
!pip install git+https://github.com/openai/whisper.git 
!pip install --upgrade --no-deps --force-reinstall git+https://github.com/openai/whisper.git

!whisper "/content/auguri_mamma.m4a" --model large-v2 --language Italian --device cuda

```
![IMG-20230213-WA0011.jpg](https://user-images.githubusercontent.com/62790059/218574759-005b1ecd-1e3d-49e9-bca5-199da0a62d99.jpg)

I thought the audio was low volume or did not feel well but listening back to it the audio was comprensible

**Comment by BabyWalking** [comment]
I have the same problem with yours. Have you find the cause root? I have no idea about this.

**Comment by glangford** [comment]
This is a common problem (search the discussions for "hallucination"). 

Try adding the command line option (note the empty double quotes)
`--suppress_tokens ""`

If that doesn't work, add
`--condition_on_previous_text False`

**Reply by BabyWalking** [reply]
Thank you , I will give a shot

**Reply by Ironship** [reply]
Did IT help?
